blockchain/blocks/src/header/mod.rs

```diff
@@ -236,7 +236,7 @@ impl BlockHeader {
     pub fn beacon_entries(&self) -> &[BeaconEntry] {
         &self.beacon_entries
     }
-    /// Getter for window PoSt proof
+    /// Getter for winning PoSt proof
     pub fn win_post_proof(&self) -> &[PoStProof] {
```

Comment:
lmao i can see why this function name can cause confusion (winning vs window)

Reply:
I mean, there is nothing to confuse it with since there is no window post on the header, but I might as well change the function name too.

This was probably from before the PoSt proof split (iirc the name of it was window? If not idk why this comment)